src/staking-v3/store/mutations.ts

```diff
@@ -19,6 +19,7 @@ import {
 export interface DappStakingMutations<S = DappStakingState> {
   addDapps(state: DappStakingState, dapps: CombinedDappInfo[]): void;
   addNewDapps(state: DappStakingState, dapps: DappInfo[]): void;
+  numberOfParticipants(state: DappStakingState, numberOfParticipants: number): void;
```

Comment:
Prefix name with verb, explaining what method is doing. I this case most likely `set`. Do the same for getters. I am not sure why I prefix all getters with `get` when this is obvious, but please add `get` prefix for consistency sake.